uptane-standard.md

```diff
@@ -74,30 +87,123 @@ This document describes a framework for securing automotive software update syst
 
 # Introduction
 
-TODO
+Uptane is a secure software update framework for automobiles. This document describes procedures to enable programmers for OEMs and suppliers to design and implement this framework to better protect connected units on cars. Integrating Uptane as outlined in the sections that follow can reduce the ability of attackers to compromise critical systems. It also assures a faster and easier recovery process should a compromise occur.
+
+These instructions specify the components necessary for a compliant implementation. Individual implementors can make their own technological choices within those requirements. This flexibility makes Uptane adaptable to the many customized update solutions used by manufacturers.
 
 # Terminology
 
 ## Conformance Terminology
 
+The keywords MUST, MUST NOT, REQUIRED, SHALL, SHALL NOT, SHOULD, SHOULD NOT, RECOMMENDED, MAY, and OPTIONAL in this document are to be  interpreted as described in {{RFC2119}}.
+
+In order to be considered “Uptane-compliant,” an implementation MUST follow all of these rules as specified in the document.
+
 ## Automotive Terminology
 
+*Bus*: An internal communications network that interconnects components within a vehicle. A car can have a number of buses that will vary in terms of power, speed and resources.
+*Image*: File containing all the relevant data and metadata for an ECU. Each ECU typically holds only one image, although this may vary in case to case.
+*Metadata*: Data about a file that can be used to verify its authenticity and currency, and to assure that it has not been tampered with or altered being uploaded to the repository. Metadata information is generally cryptographic hashes and file lengths.
```

Comment:
"Metadata information is generally cryptographic hashes and file lengths." -> Maybe better as "Metadata information generally includes cryptographic hashes and file lengths."

Reply:
I'd be inclined to say that the whole definition should be reworked. Vehicle manifests are certainly metadata, for example, and don't fit this definition. There also appears to be a word missing -> "[...] tampered with or altered <?after?> being uploaded [...]".

Reply:
Actually, on second thought: I think we should remove this definition altogether. The true definition of metadata is simply "data about data", and I think it's safe to assume that any reader of this standard will know that definition. The fact that Uptane uses metadata to ensure authenticity of data, and that Uptane metadata contains hashes and file sizes is relevant, but it's not part of the definition of metadata.

Reply:
Resolved with e063f0db307af208dcff6cf09bda79b9bf17a4e1